src/main/java/org/prebid/server/bidder/ix/model/response/IxBidResponse.java

```diff
@@ -0,0 +1,26 @@
+package org.prebid.server.bidder.ix.model.response;
+
+import com.iab.openrtb.response.SeatBid;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Builder(toBuilder = true)
+@Value
+public class IxBidResponse {
```

Comment:
Why add new class when you can just unmarshall bidRequest.ext to IxExtBidResponse?

Reply:
Just tried to do in the same style as other bidders. If option, described by you, is ok - I may do this. Not a big deal, I think.

Reply:
Looks like done. Should work as expected, if IX will use response from ORTB spec.